### PR TITLE
add semantic ui stylesheet and components (Header, Main)

### PR DESCRIPTION
**Pull request by rpeyfuss**
@davidwparker @willklein @natevenn 
create style/css folder in public folder
add --content-base public/ to start script in package.json
add jQuery dependency in package.json
add css loader to webpack.config.js
create Header component
create Main component

[link to Semantic UI](http://semantic-ui.com/)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "babel-loader": "^6.2.0",
     "babel-preset-es2015": "^6.3.13",
     "babel-preset-react": "^6.3.13",
-    "webpack-dev-server": "^1.14.0"
+    "webpack-dev-server": "^1.14.0",
+    "jquery": "^2.2.4"
```
Comment (davidwparker) [inline package.json]: Reason for jquery?

Comment (rpeyfuss) [inline package.json]: will remove jquery; 
originally added the semantic ui js file, which needed jquery, but then removed it;

**Comment by rpeyfuss** [comment]
@willklein @davidwparker @natevenn 
Can I merge?

**Comment by davidwparker** [comment]
Can you add to .gitignore:

```
.DS_Store
public/bundle.js  (or maybe all of public it's just being generated)?
```

Can we add semantic-ui via npm rather than manually? 
https://www.npmjs.com/package/semantic-ui

**Comment by natevenn** [comment]
@davidwparker if I npm install semantic-ui it adds a whole bunch of files that look like this
├── semantic
│   ├── dist
│   ├── gulpfile.js
│   ├── src
│   └── tasks
├── semantic.json
├── src
│   ├── components
│   ├── config.js
│   ├── containers
│   ├── decorators
│   ├── helpers
│   ├── index.html
│   ├── index.js
│   ├── redux
│   ├── routes.js

where would you recommend that I dump these files? in the root?

File: public/index.html [hunk 2]
```diff
@@ -3,9 +3,10 @@
 <head>
   <meta charset="UTF-8">
   <title>ReactJS-Denver</title>
-  <link rel="stylesheet" type="text/css" href="/style/css/semantic.min.css">
-  <link rel="stylesheet" type="text/css" href="/style/css/style.css">
-  </head>
+  <script type="text/javascript" src="//code.jquery.com/jquery-1.10.0.min.js"></script>
```
Comment (natevenn) [inline public/index.html:6]: semantic-ui depends on jquery

**Comment by natevenn** [comment]
@willklein @davidwparker @rpeyfuss 
Quickest and easiest solution for not including the semantic-ui.min.css in the app. I just used the semantic cdn which also depends on jquery. I also npm installed react-semantify to help integrate semanitc-ui with react components https://github.com/jessy1092/react-semantify

**Pull request merged**